src/core/segment_reader.rs

```diff
@@ -128,7 +128,8 @@ impl SegmentReader {
         })
     }
 
-    pub(crate) fn fieldnorms_readers(&self) -> &FieldNormReaders {
+    #[doc(hidden)]
+    pub fn fieldnorms_readers(&self) -> &FieldNormReaders {
```

Comment:
Is `get_fieldnorms_reader` not sufficent?

Reply:
I need to access the file slice to download relevant parts asynchronously. Unfortunately, I can't use `FieldNormReader` for that.